### DISCUSSION TITLE
绳网的服务器怎么进？（右上角的那个）

### DISCUSSION THREAD

**Discussion by 206288602** [share121/inter-knot#2453] [其他]
咱一直进不去<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by nostalgiatan** [comment]
那说明你的以太适应性不强，咱不强求

**Comment by HifumiMizuhara** [comment]
Discord？这个区域以后再来探索吧

**Comment by Qweulin** [comment]
也是国外网站/app，自行摸索

**Comment by cpu644** [comment]
使用加速器。国外网站
